srcpkgs/openimagedenoise/template

```diff
@@ -0,0 +1,26 @@
+# Template file for 'openimagedenoise'
+pkgname=openimagedenoise
+version=1.1.0
+revision=1
+archs="x86_64*"
+wrksrc=oidn-${version}
+build_style=cmake
+hostmakedepends="python3"
+makedepends="tbb-devel"
+depends="python3"
```

Comment:
I think it's not necessary at run-time?

Reply:
right, it's only required at build time. You can remove this `depends` line.

Reply:
done